### ISSUE TITLE
New install - Server Crash on startup

### ISSUE THREAD

**Issue by garyoneillj** [AllTheMods/ATM-8#1535]
Tried figuring this out but to no avail.  Played plenty of modpacks before and plenty of troubleshooting experience.  But I'm just lost as to what to do here.  Tried using multiple older server versions also and getting the same error.

[Server-Files-1.0.20b Crash on startup.txt](https://github.com/AllTheMods/ATM-8/files/11979989/Server-Files-1.0.20b.Crash.on.startup.txt)

**Comment by Mitchell5200** [comment]
Use java 17, not 20

**Comment by Duros001** [comment]
Same Issue, and I'm using Java 17

**Comment by Mitchell5200** [comment]
> Same Issue, and I'm using Java 17

Logs please, via https://mclo.gs/, upload, then paste the link back here.

**Comment by garyoneillj** [comment]
In the serverstart.bat I saw this line:
:: To use a specific Java runtime, set an environment variable named ATM8_JAVA to the full path of java.exe.

After adding that argument directly pointing to the Java 17 java.exe my server worked first time.  So whilst I did have Java 17, and Java 20, it was defaulting to 20.  So for me the answer was indeed to use Java 17, but had to be specified there.

For me the issue is solved, unsure if @Duros001 has the same issue or not.
